README.md

```diff
@@ -182,6 +182,7 @@ This endpoint is used to send a batch of messages. It accepts a JSON payload con
 - **Headers:**
   - Authorization: `Bearer <CLIENT_TOKEN>`
   - Content-Type: application/json
+  - Consumer-Key: `some-consumer`
```

Comment:
Not a dealbreaker but non-standard headers are often prefixed `x-` https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers _Custom proprietary headers have historically been used with an X- prefix, but this convention was deprecated in 2012_. [The headers we expect from Notify are prefixed this way](https://github.com/NHSDigital/dtos-communication-management/blob/main/src/notify/app/validators/request_validator.py#L7-L8) so we might want to be consistent.

Reply:
ah yeah I meant to revisit this, as I understood that `x-` was deprecated - happy to just follow existing convention though